src/modules/reporting/components/reporting-dispute-form/reporting-dispute-form.jsx

```diff
@@ -61,166 +61,197 @@ const validateNumber = (validations, updateState, fieldName, value, humanName, m
   })
 }
 
-const validateStake = (validations, updateState, isStakeRequired, stake) => {
-  const updatedValidations = { ...validations }
+const validateStake = (p, stake) => {
+  const updatedValidations = { ...p.validations }
 
-  if (isStakeRequired) {
+  if (p.stakeIsRequired) {
     switch (true) {
       case stake === '':
-        updatedValidations.stake = `The stake field is required.`
+        updatedValidations.stake = 'The stake field is required.'
         break
-      case stake <= 0:
-        updatedValidations.stake = `Please enter a stake greater than 0.`
+      case parseFloat(stake) <= parseFloat(p.minimumStakeRequired):
```

Comment:
Things like `parseFloat` should never be used in the UI due to potential routing errors — under the assumption that all numbers in the UI are of BigNumber format, you can use it’s helper methods to format for display.

This is not the case currently due to mocked data, but it’s definitely something to be aware of going forward.